e2e/pages/caseView.page.js

```diff
@@ -30,9 +30,8 @@ module.exports = {
   caseTitle: '.case-title .markdown',
 
   async goToNewActions(actionSelected) {
-    I.waitForElement(this.actionsDropdown);
```

Comment:
this throws exception if for some reason component is not found within default timeout (20s)